Documentation/design/resource_dep.dot

```diff
@@ -0,0 +1,185 @@
+    operator_manifests [label="operator manifests"];
+  }
+
+  // Cluster assets
+  subgraph cluster_cluster_assets {
+    style=invis;
+    node [style=filled, color=lightpink];
+    bootstrap_control_plane [label="bootstrap control plane manifests"];
+    kubernetes_control_plane [label="kubernetes control plane manifests"];
+  }
+
+  // Ignition files
+  subgraph cluster_ignition {
+    style=invis;
+    node [style=filled, color=gold];
+    bootstrap_ignition [label="bootstrap.ign"];
+    master_ignition [label="master.ign"];
+    worker_ignition [label="worker.ign"];
+  }
+
+  // Kubeconfigs
+  subgraph cluster_kubeconfig {
+    style=invis;
+    node [style=filled, color=lightblue];
+    kubelet_kubeconfig [label="kubeconfig-kubelet"];
+    kubeconfig [label="kubeconfig",shape=box,style=filled,color=red,fillcolor=lightblue,penwidth=3];
+  }
+
+
+  // Output after cluster launch
+  {
+    node [shape=box,style=filled,color=red,fillcolor=yellowgreen,penwidth=3];
+    terraform_state [label="*.tfstate"];
+  }
+
+  // Dependencies
+  password -> install_config;
+  email_address -> install_config;
+  pull_secret -> install_config;
+  license -> install_config;
+  cluster_name -> install_config;
+  base_domain -> install_config;
+
+  install_config -> operator_manifests;
+  install_config -> operator_configs;
+  install_config -> admin_bundle
+  install_config -> aggregator_ca_bundle;
+  install_config -> apiserver_bundle;
+  install_config -> apiserver_proxy_bundle;
+  install_config -> cluster_apiserver_bundle;
+  install_config -> etcd_ca_bundle;
+  install_config -> etcd_client_bundle;
+  install_config -> ingress_ca_bundle;
+  install_config -> ingress_bundle;
+  install_config -> kube_ca_bundle;
+  install_config -> kubelet_bundle;
+  install_config -> openshift_apiserver_bundle;
+  install_config -> root_ca_bundle;
+  install_config -> service_account_bundle;
+  install_config -> service_serving_ca_bundle;
+  install_config -> mcc_bundle;
+  
+
+  admin_bundle -> kubeconfig;
+
+  aggregator_ca_bundle -> bootstrap_ignition;
+  aggregator_ca_bundle -> kubernetes_control_plane; // kube-apiserve-secret, openshift-apiserver-secret.
+
+  apiserver_bundle -> bootstrap_ignition;
+  apiserver_bundle -> kubernetes_control_plane; // kube-apiserve-secret, openshift-apiserver-secret.
+
+  apiserver_proxy_bundle -> bootstrap_ignition;
+  apiserver_proxy_bundle -> kubernetes_control_plane; // kube-apiserve-secret, openshift-apiserver-secret.
+
+  cluster_apiserver_bundle -> bootstrap_ignition;
+  cluster_apiserver_bundle -> kubernetes_control_plane; // cluster-apiserver-secret.
+
+  etcd_ca_bundle -> kubernetes_control_plane;
+  etcd_ca_bundle -> master_ignition;
+
+  etcd_client_bundle -> kubernetes_control_plane;
+  etcd_client_bundle -> master_ignition;
+
+  ingress_ca_bundle -> kubernetes_control_plane;
+  ingress_ca_bundle -> bootstrap_ignition;
+
+  kube_ca_bundle -> 
+
+  root_ca_bundle -> bootstrap_ignition;
```

Comment:
You may want to remove the blank line here and just have:

```dot
kube_ca_bundle -> root_ca_bundle -> bootstrap_ignition;
```

Although I'd expect the `root_ca_bundle` would be used to sign the `kube_ca_bundle` (and therefore be an ancestor of it instead of being a descendant).  Perhaps you meant:

```dot
kube_ca_bundle -> kubernetes_control_plane;
```